services/github/src/utils.js

```diff
@@ -212,14 +219,20 @@ export const checkGithubSignature = (secret, payload, signature) => {
   return compareSignatures(signature, expected);
 };
 
+export const buildUrl = (taskId, runId, artifactName) => {
+  return `${process.env.TASKCLUSTER_ROOT_URL}/tasks/${taskId}/runs/${runId}/${artifactName}`;
```

Comment:
let's not use `process.env` here and just explicitly pass `rootUrl` as we do in https://github.com/taskcluster/taskcluster/blob/main/services/github/src/handlers/utils.js#L9-L10 

Probably also makes sense to place this function close to the other similar ones?

Reply:
Have incorporated the change of not using process.env, didn't get the question of placing function to similar ones, do we mean to keep it in utils inside handlers?